frontend/packages/pipelines-plugin/src/components/quicksearch/PipelineQuickSearchDetails.tsx

```diff
@@ -38,35 +39,6 @@ const PipelineQuickSearchDetails: React.FC<QuickSearchDetailsRendererProps> = ({
   const [ctaType, setCtaType] = React.useState<string>();
   const [buttonText, setButtonText] = React.useState<string>();
```

Comment:
This was in a different PR... But it is odd that this is calculated through a `useEffect`. Why not just a variable?